extensions/discourse_tagging.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module ::DiscourseTagging
+  class << self
+    alias_method :discourse_core_filter_allowed_tags, :filter_allowed_tags
+    def filter_allowed_tags(guardian, opts = {})
+      if tagGroups = RequestStore.store[:tagGroups]
```

Comment:
@fzngagan We always use ``underscore_case`` for variables in ruby.

Reply:
Noted.

Reply:
Fixed.